binder/src/main/scala/PanamaCIRCTConverter.scala

```diff
@@ -781,10 +781,12 @@ class PanamaCIRCTConverter extends CIRCTConverter {
 
     val builder = util
       .OpBuilder("firrtl.extmodule", firCtx.circuitBlock, circt.unkLoc)
+      .withRegionNoBlock()
```

Comment:
This line fixed the assertion at https://github.com/llvm/circt/blob/29ba8077af794dac78b90a8707da376489c398fe/lib/Dialect/FIRRTL/Transforms/LowerOpenAggs.cpp#L318